docs/update-design-question.md

```diff
@@ -0,0 +1,45 @@
+Guys, we have a question for Formica where everyone is invited to participate deciding.
+
+So, formica currently is a tool for managing groups of units. Example:
+
+```
+$ formicactl submit elasticsearch@{1,2}
+$ formciactl start elasticsearch@{1,2}
+$ formicactl status elasticsearch
+Group           Unit  <more headers>
+elasticsearch@1 *     active  active  running 10.0.4.100
+elasticsearch@2 *     active  active  running 10.0.4.101
+```
+
+As you see, it is very similar to `fleetctl`. Next we want to implement an update
+functionality, similar (but better ;) than what `releaseit` currently does.
+
+## Question 1
+In the example above we have used `1` and `2` for the slices. When updating, we
+want the users to have the option to create new instances first. This raises where
+the names for those slices come from.
+
+* Option 1: Just increment the found ids. So if we find `7,8,9` we continue with
```

Comment:
This part is an identifier for the instance. Incrementing is probably not a good idea. Just think of two people working with the system at the same time. If you think of higher level orchestration it would be helpful to receive that instance identifier (eg. our instance id or similar). 

I took a quick look at how `systemd-run` handles this. This is not a perfect match but you can define a `--unit` or it creates a random one.